modules/registry-third-party-registries.adoc

```diff
@@ -2,15 +2,36 @@
 //
 // * registry/index.adoc
 
+:_content-type: PROCEDURE
 [id="registry-third-party-registries_{context}"]
 = Third-party registries
 
-{product-title} can create containers using images from third-party registries,
-but it is unlikely that these registries offer the same image notification
-support as the integrated {product-title} registry. In this situation,
-{product-title} will fetch tags from the remote registry upon imagestream
-creation.
+{product-title} can create containers using images from third-party registries, but it is unlikely that these registries offer the same image notification support as the integrated {product-title} registry. In this situation, {product-title} will fetch tags from the remote registry upon imagestream creation. Refreshing the fetched tags is as simple as running `oc import-image <stream>`. When new images are detected, the previously described build and deployment reactions occur.
```

Comment:
`{product-title} will fetch tags from the remote registry upon imagestream creation. Refreshing the fetched tags is as simple as running `oc import-image <stream>`. `
This confuses me. Does the user fetch the tags with the command or OpenShift?

Reply:
OpenShift fetches the tags and the user can refresh the fetched tags using `oc import-image <stream>` command.

Reply:
Changed the sentence to " To refresh the fetched tags, run oc import-image <stream>."